Jenkinsfile_CNP

```diff
@@ -21,8 +21,8 @@ so for testing purposes it makes sense to use as an example.
  */
 def nonPrSecrets = [
   'bulk-scan-${env}': [
-    secret('notifications-queue-listen-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_READ'),
-    secret('notifications-queue-send-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_WRITE')
+    secret('notifications-staging-queue-listen-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_READ'),
```

Comment:
it should be reform-scan ?

Reply:
oh my lawd!

Reply:
and  did we do any code change to use staging queue?

Reply:
oh yes @mdayican hasn't done the change yet for using staging so we will need to use the normal queue for functional tests. Good point Mustafa. Also please ensure secrets in vault has correct value.